src/views/DeleteAccount.vue

```diff
@@ -0,0 +1,146 @@
+<template>
+    <b-form id="login-container">
+        <a class="close-button" @click="back"></a>
+        <div class="logo" />
+        
+        <p style="font-weight: bold">Are you sure you want to delete your account?</p>
+        <p>
+            We're sorry to hear that you're leaving us! Something wrong that we can help with?
+            Please get in touch at <a href="mailto:support@eternagame.org">support@eternagame.org</a>.
+        </p>
+        <p>
+            Your account will be deactivated and scheduled to be deleted in 30 days.
+            Per our privacy policy found at <a href="https://eternagame.org/terms">https://eternagame.org/terms</a>,
+            your identifying information will be removed and your contributions will be anonymized
+            (though we cannot retroactively remove names that may have been included in
+            scientific publications). If you have used our forum, you will also need to separately
+            delete your forum account at <a href="https://forum.eternagame.org/">https://forum.eternagame.org/</a>.
+        </p>
+        <p>
+            Thank you for joining us on our mission to advance science! To confirm you want to delete
+            your account, please enter your username and password below and click delete.
+        </p>
+        
+        <b-form-input type="text" size="md" :disabled="isLoading" v-model="username" placeholder="username"></b-form-input>
+        <b-form-input type="password" size="md" :disabled="isLoading" v-model="password" placeholder="password"></b-form-input>
+        <div class="alert-container">
+            <b-alert v-if="error" variant="danger" dismissable>
```

Comment:
This alert doesn't appear. I checked Bootstrap Vue and I think `b-alert` expects a show prop rather than v-if, e.g. `<b-alert :show=" error !== '' ">`.

Reply:
The way we have it in the frontend wherever we use it appears to be `<b-alert show v-if="...">` - I think we do it that way because we wrap it in a `<transition>` to have it fade in. I might want to implement that

Reply:
Actually, I'll pass on that at least for now since we don't do it elsewhere. I'll just use `:show`